javascript-algorithms-and-data-structures/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/index.js

```diff
@@ -24,108 +18,113 @@ const categories = {
   560: { category: "Backend Development", className: "backend" },
 };
 
-const forumCategories = (id) => {
-  return categories.hasOwnProperty(id)
-    ? `<a class="category ${categories[id]["className"]}" target="_blank" href="${FORUM_CATEGORY_URL}${categories[id]["className"]}/${id}">${categories[id]["category"]}</a>`
-    : `<a class="category general" target="_blank" href="${FORUM_CATEGORY_URL}general/1">General</a>`;
-};
+async function fetchData() {
+  const res = await fetch(FORUM_LATEST);
+  const data = await res.json();
+  showLatestPosts(data);
+}
+fetchData();
 
-const showTime = (time) => {
-  const lastPost = new Date(time);
-  const currentTime = new Date();
+const forumCategory = (id) => {
+  let selectedCategory = {};
 
-  if (currentTime.getUTCHours() === lastPost.getUTCHours()) {
-    return `${currentTime.getMinutes() - lastPost.getUTCMinutes()}m`;
+  if (allCategories.hasOwnProperty(id)) {
+    const { category, className } = allCategories[id];
+    selectedCategory.className = className;
+    selectedCategory.category = category;
   } else {
-    return `${currentTime.getHours() - lastPost.getHours()}h`;
+    selectedCategory.className = "General";
```

Comment:
If we lowercase the class selector in `styles.css`, this should be lowercased, too:

```suggestion
    selectedCategory.className = "general";
```